app/controllers/map_location_controller.rb

```diff
@@ -23,7 +23,7 @@ def dul_location_service(collection_code, sublibrary, callno)
   end
 
   def map_uri
-    @map_uri ||= URI.parse('https://library.duke.edu/locguide/mapinfo')
+    @map_uri ||= URI.parse(ENV['MAP_LOCATION_SERVICE_URL'])
```

Comment:
Reference to configured url should be `DulArgonSkin.map_location_service_url`

Reply:
Good to know - thanks again!!